crates/table/src/pointer_map.rs

```diff
@@ -234,8 +237,6 @@ impl PointerMap {
     ///
     /// Handles any hash conflicts for `hash`.
     pub fn insert(&mut self, hash: RowHash, ptr: RowPointer) -> bool {
-        debug_assert!(self.maintains_invariants());
```

Comment:
I'm not sure why these were left in, but they make populating a table quadratic time.